fluent-package/test-repack-package.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/bash
+
+#
+# Usage: test-repack-package.sh [OPTIONS] PATH_TO_DEB_OR_RPM
+#
+# e.g.
+# $ test-repack-package.sh debug fluent-package-6.0.0-1.el8.x86_64.rpm
+# $ test-repack-package.sh debug fluent-package_6.0.0-1_amd64.deb
+#
```

Comment:
Thanks for this!
It looks convenient, but I don't understand what we can do by using this script...
Could you please reinforce these usage comments?

Reply:
Thanks, I've updated explanation